modules/core/src/test/scala/org/scalasteward/core/mock/MockConfig.scala

```diff
@@ -5,7 +5,7 @@ import org.scalasteward.core.application.Cli.ParseResult.Success
 import org.scalasteward.core.application.{Cli, Config}
 
 object MockConfig {
-  val mockRoot: File = File.temp / "scala-steward"
+  val mockRoot: File = File.newTemporaryDirectory() / "scala-steward"
```

Comment:
> There is some kind of cleanup in the sbt build:
> 
> Too bad this is ignored by IntelliJ.

Ah, interesting, that's two things I wasn't aware of- that there was test cleanup code in Scala Steward's build.sbt, and also that IntelliJ wouldn't _run_ that test cleanup code (I've reported that as [SCL-21353](https://youtrack.jetbrains.com/issue/SCL-21353/Scala-IntelliJ-test-runner-doesnt-perform-sbt-Tests.Setup-or-Tests.Cleanup-defined-in-testOptions?s=Scala-IntelliJ-test-runner-doesnt-perform-sbt-Tests.Setup-or-Tests.Cleanup-defined-in-testOptions) against IntelliJ). I think the _discoverability_ of those two bits of information is an issue for newcomers trying to contribute to this project - how can we help them?

There are a couple of options for helping out newcomers here:

### A. Inform IntelliJ users that they _must_ run the tests via `sbt`

Although the information could be added to [CONTRIBUTING.md](https://github.com/scala-steward-org/scala-steward/blob/main/docs/CONTRIBUTING.md), it probably stands a better chance of attracting the developers attention if it's displayed at the most pertinent point, ie when the developer runs the tests, like this: 

```suggestion
  val mockRoot: File = File.temp / "scala-steward"
  require(mockRoot.isEmpty,
    s"""Please ensure that the following folder is *empty*
       |(note that build.sbt `testOptions` is set to auto-cleanup contents, but
       |running tests in IDEs like IntelliJ may not execute the `CleanUp`):
       |$mockRoot""".stripMargin
  )
```

### B. Change test clean-up so IntelliJ works without contributors needing to adapt

> We could also change the config to be created within an MUnit Fixture, so it can be properly teardown.

Personally, that sounds great to me - an even smaller change (than an MUnit fixture) would be to add 1 line to perform the deletion in `MockConfig`: 

```suggestion
  val mockRoot: File = File.temp / "scala-steward"
  mockRoot.delete() // Ensure folder is cleared of previous test files
```

How does that sound to you?

Reply:
👍 for plan B. I guess we could then remove the cleanup in the build too?

Reply:
Yeah, sounds good - I've amended the commit to go for option B, and also removed the corresponding sbt configuration 👍  - new commit is f44e79df92008a39c59b6b3627b686ee1683b8bd.